doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,260 @@
+{
```

Comment:
<p>I think that&nbsp;<code>SimilarityExplainer</code>&nbsp;should be replaced everywhere by&nbsp;<code>GradientSimilarity</code></p><p>of the type "I classify this .. " -&gt; of the type: "I classify this ... "</p><p>introduced by&nbsp;<a href="https://papers.nips.cc/paper/2019/hash/c61f571dbd2fb949d3fe5ae1608dd48b-Abstract.html" target="_blank">Charpiat et al., 2019</a>&nbsp;-&gt;&nbsp;introduced by&nbsp;<a href="https://papers.nips.cc/paper/2019/hash/c61f571dbd2fb949d3fe5ae1608dd48b-Abstract.html" target="_blank">Charpiat et al., 2019</a>. (add dot)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
@gipster the rename has only been done partially